docs/02-sygma-protocol/02-Tailored-Security/03-Spectre/01-spectre-intro.md

```diff
@@ -0,0 +1,67 @@
+---
+slug: /tailoredsecurity/spectre/intro
+id: tailoredsecurity-spectre-intro
+title: Introduction To Spectre
+description: The following details how Spectre is utilized by Sygma.
+draft: false
+---
+
+:::info
+The following details Sygma's Spectre verification system utilizing zero-knowledge (zk) proofs of consensus between EVM domains. Spectre was developed by ChainSafe as a set of zk cryptographic provers, arithmetic circuits, and contracts. Protocol Sygma integrates Spectre, along with custom infrastructure built to utilize Spectre, to provide zk verification. For ease of definition, this verification system will be generally referred to as "Spectre", but there are distinctions between components built for Sygma and cryptographic components built for Spectre.
```

Comment:
Should we also talk here about the coming storage proofs and the importance of proofing both? I think it's important!

Reply:
I agree @alexmmueller, but this should, in my opinion, go to a separate PR. Can you, @haochizzle, create simple issues for some of these topics that we want to add or improve?

Reply:
added as an issue #148